packages/smooth_app/lib/database/product_query.dart

```diff
@@ -43,14 +42,12 @@ abstract class ProductQuery {
       '_'
       '${getCountry()!.iso2Code.toUpperCase()}';
 
-  /// Device Id, potentially used as API uuid.
-  static String? deviceId;
   static const String UUID_NAME = 'UUID_NAME';
 
   /// Sets the device id as "final variable", for instance for API queries.
   ///
   /// To be called at main / init.
-  static Future<void> setDeviceId(final LocalDatabase _localDatabase) async {
+  static Future<void> setUuid(final LocalDatabase _localDatabase) async {
```

Comment:
A file is missing, right? The one that called `setDeviceId`...

Reply:
just noticed that too